src/einsteinpy/tests/test_plotting/test_scattergeodesicplotter.py

```diff
@@ -52,3 +52,16 @@ def test_plot_show_shows_plot(mock_show):
     cl.plot(r, v, el, ss)
     cl.show()
     mock_show.assert_called_with()
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.ScatterGeodesicPlotter.save")
```

Comment:
I think you didn't understand wat to mock. I meant you can mock:

`einsteinpy.plotting.geodesics_static.plt.savefig`

Reply:
Ohh! Sorry, my bad.

Reply:
@shreyasbapat Could you please help me exactly how to do the mocking? Also, where should I put the parameters for `savefig`?

Reply:
Oh! Sure. You can take help from the documentation: https://docs.python.org/3/library/unittest.mock-examples.html#patch-decorators